source/style-guide/information-types.txt

```diff
@@ -0,0 +1,75 @@
+.. _information-types:
+
+=================
+Information Types
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+The following sections describe information typing and provide
+prototypes that show how to apply this strategy to our documentation.
+
+What is Information Typing?
+---------------------------
+
+Information typing categorizes content by type based on your audience's
+goals. Throughout the writing process, information typing helps keep the
+information that you write focused on those goals.
+
+Additionally, information typing accomplishes the following goals:
+
+- Separate content into categories and delivers roughly one type at a
+  time.
+
+- Account for the purpose of a page before you write it.
+
+- Answer the user question "why do I care?" or "what does this mean for
+  me?" quickly.
+
+- Use a suggested format to help you get started on a blank page.
+
+What Information Types Do We Use?
```

Comment:
Just to make sure I understand, is the request to surface similarities and differences between our information types and DITA's information types, or between the individual types themselves? (As in, similarities / differences between Task and Concept, for example).

Reply:
I meant the similarities and differences between these types and the DITA descriptions linked in the prior comment. In the DITA types, the types are meant to be distinct and I'm assuming that's the intent here too. 
However, if my assumption is incorrect, that would be unexpected, and would want to know what the overlap is and reasons for that.

In terms of surfacing differences with DITA, here are a couple potential solutions, without making assumptions about whether they're supposed to be similar or different:
- Mention that these types are similar to the ones found in DITA. Explain any differences.
- Mention that these types are not similar to the ones found in DITA. Provide reasons why these types work for us better than the ones found in DITA.

Let me know if that makes sense. Feel free to reach out to me on Slack otherwise!

Reply:
Got it, thanks for clarifying! I added a small paragraph with a link to the DITA descriptions. Do you think something like what I've added [here](https://preview-mongodbjeffallenmongo.gatsbyjs.io/meta/concept-page-prototype-updates/style-guide/information-types/index/#comparison-with-dita-document-types) works?

Reply:
I think the content is good, but maybe it could be presented before in a note admonition prior to the description of the types. I think audiences that might be familiar with DITA already would be able to experience the content that follows it with that additional knowledge, whereas when presented after the content, they might need to backtrack.

Reply:
Thanks for this suggestion. Agreed on placement. I moved it up above the table.